packages/topiary/topiary.0.6.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+
+maintainer: "hello@tweag.io"
+authors: [ "Tweag" ]
+
+homepage: "https://topiary.tweag.io/"
+bug-reports: "https://github.com/tweag/topiary/issues"
+dev-repo: "git+https://github.com/tweag/topiary.git"
+
+license: "MIT"
+depends: ["conf-rust-2021"]
```

Comment:
```suggestion
depends: ["conf-rust-2024"]
```
Once https://github.com/ocaml/opam-repository/pull/27376 is merged

This way we may be able to provide better depexts

Reply:
Sounds reasonable!